monkey/monkey_island/cc/services/config.py

```diff
@@ -443,3 +445,33 @@ def _remove_credentials_from_flat_config(config: Dict):
 
         for field in fields_to_remove:
             config.pop(field, None)
+
+    @staticmethod
+    def _format_payloads_from_flat_config(config: Dict):
+        config.setdefault("payloads", {})["ransomware"] = config["ransomware"]
+        config.pop("ransomware", None)
+
+    @staticmethod
+    def _format_pbas_from_flat_config(config: Dict):
+        old_linux_command = "custom_PBA_linux_cmd"
+        old_linux_filename = "PBA_linux_filename"
+        old_windows_command = "custom_PBA_windows_cmd"
+        old_windows_filename = "PBA_windows_filename"
```

Comment:
They are not "old". `linux_command_key` and `windows_command_key` would better reflect that we're retrieving values from flat config to form a new dict in a different format